packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr-create-cluster.ts

```diff
@@ -0,0 +1,1452 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import { getResourceArn } from './resource-arn-suffix';
+
+/**
+ * Properties for EmrCreateCluster
+ *
+ * See the RunJobFlow API for complete documentation on input parameters
+ *
+ * @see https://docs.aws.amazon.com/emr/latest/APIReference/API_RunJobFlow.html
```

Comment:
Is there a reason you left out the `Steps` parameter of the RunJobFlow API?

If I'm understanding correctly, with this change we would have to create the cluster and add steps as separate Sfn tasks. Ideally, we would be able to create the cluster with a list of steps in one task and SYNC until all execution is done.

Reply:
Steps were intentionally left out. See the documentation on `createCluster` for the Step Functions/EMR integrations here: https://docs.aws.amazon.com/step-functions/latest/dg/connect-emr.html

According to the docs, Steps is invalid for this integration. For transient clusters, the intent is to use the State Machine to create the cluster, add and monitor Steps, then terminate the cluster. The `addStep` integration can synchronously wait for Step completion, this allows the State Machine to react to success or failure of individual Steps.

The `addStep` can also be used to add and monitor Steps on persistent clusters.